python3/deploy

```diff
@@ -8,11 +8,14 @@ SOURCE_DIR=/var/lib/tsuru
 source ${SOURCE_DIR}/base/deploy
 source ${SOURCE_DIR}/base/rc/config
 
+APP_VENV=/home/application/.app_env
+source $APP_VENV/bin/activate
```

Comment:
I believe that to use pip from virtualenv is a better option than to do the activate